util/c-compress-test.pl

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env perl
+#
+# TEST c-compress-pl with a number of examples and what should happen to them
```

Comment:
shouldn't this be in the test directory?

Reply:
We have no tests of our internal utils, so not until we figure something out.  In the mean time, this exists to verify that `c-compress.pl` works as advertised...